API/NoDocs/Microsoft.AspNetCore.Testing/ServiceFakesExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Assembly 'Microsoft.AspNetCore.Testing'
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.AspNetCore.Testing;
+
+public static class ServiceFakesExtensions
+{
+    public static IWebHostBuilder UseFakeStartup(this IWebHostBuilder builder);
```

Comment:
What does ths do?

Reply:
@tekian Do you have the background here?

Reply:
This
```
        using var host = FakeHost.CreateBuilder()
            .ConfigureWebHost(webHost => webHost.UseFakeStartup().ListenHttpOnAnyPort())
            .Build();
```
prevents this exception:
```
System.InvalidOperationException: No application configured. Please specify an application via IWebHostBuilder.UseStartup, IWebHostBuilder.Configure, or specifying the startup assembly via StartupAssemblyKey in the web host configuration.
    at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
    at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__15_1(IHostedService service, CancellationToken token)
    at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
    at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
    at Microsoft.Extensions.Hosting.Testing.FakeHost.StartAsync(CancellationToken cancellationToken) in D:\git\extensions\src\Libraries\Microsoft.Extensions.Hosting.Testing\FakeHost.cs:line 75
```

We could also call it `UseFakeApp`.